gsi/hnswlib_test/gxl.py

```diff
@@ -0,0 +1,76 @@
+import hnswlib
+import numpy as np
+import pandas as pd
+import datetime
+import os
+
+k = 10
+dim = 96
+ef_search = [64, 128, 256, 512]
```

Comment:
please add a comment right above summarizing what "ef_search" means and why we chose this range